gambit/1.0.0/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash 
+
+set -euo pipefail
+
+FASTA=GCF_000240185.1_ASM24018v2_genomic.fna.gz
+FASTA_URL=https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/240/185/GCF_000240185.1_ASM24018v2/$FASTA
+
+gambit --version
+
+wget $FASTA_URL
+gambit query $FASTA -o test_output.csv
+grep "Klebsiella pneumoniae" test_output.csv
```

Comment:
Note: This is the same test as version 0.5, but it's been moved to its own bash script. 

This script
1. downloads a fasta file with wget
2. runs gambit on the fasta file
3. checks if Klebsiella is in test_output.csv

The resulting line is 
```
GCF_000240185.1_ASM24018v2_genomic,Klebsiella pneumoniae,species,573,0.4691092073917389,0.0,[GCF_000240185.1] Klebsiella pneumoniae subsp. pneumoniae HS11286 (enterobacteria),,,,
```